controller/system_backup_controller.go

```diff
@@ -731,6 +731,11 @@ func (c *SystemBackupController) backupVolumesAlways(systemBackup *longhorn.Syst
 
 	volumeBackups := make(map[string]*longhorn.Backup, len(volumes))
 	for _, volume := range volumes {
+		if volume.Status.IsStandby {
+			c.logger.Infof("Skip backup for standby volume %v", volume.Name)
+			continue
+		}
```

Comment:
Does it mean user need to create standby volumes by themselves after restoring the system backup?

Reply:
No, the system backup will skip creating a backup for the DR volume. But system restore will still create the DR volume with the URL in the `FromBackup`.